package.json

```diff
@@ -34,5 +36,9 @@
   "bugs": {
     "url": "https://github.com/Landoop/kafka-connect-ui/issues"
   },
-  "homepage": "https://github.com/Landoop/kafka-connect-ui#readme"
+  "homepage": "https://github.com/Landoop/kafka-connect-ui#readme",
+  "resolutions": {
+    "graceful-fs": "^4.2.4"
+  }
```

Comment:
I see you already upgraded to Gulp version 4. From the Documentation, I see that resolutions are only needed if you want to work with Node > 12 and Gilp 3.x.x. Can you please try to remove it and see what happens?? Thanks for the PR.. 🚀

Reply:
Sure, thanks for pointing it out. The build works as expected.

Reply:
Thank you. Will merge it soon!! And sorry for the delay in getting back to you.

Reply:
Awesome, thanks a lot :)